### PR TITLE
Updating room properties

### PR DESCRIPTION
**Pull request by adapap**
This PR can serve as a starting point for updating room properties (#51). Feel free to expand on this concept.
- Update room title/description from editable fields in room
- Change `room` to be a computed property which returns `this.$store.state.room`

### CODE REVIEW

**Comment by dyc3** [comment]
@bsalgado98 Since you're assigned to the relevant issue, you should take a look at this

**Comment by dyc3** [comment]
Here's some screenshots of what the changes actually look like.
2019-09-21-220407_857x302_scrot
2019-09-21-220441_897x328_scrot

**Comment by dyc3** [comment]
I think the connection indicator should use the colors in the vuetify theme. The orange for "Connecting..." and the blue for "Connected"

File: src/views/Room.vue [hunk 1]
```diff
@@ -1,9 +1,12 @@
 <template>
   <div>
     <v-container class="room" v-if="!showJoinFailOverlay">
-      <v-layout column>
-        <h1>{{ $store.state.room.title != "" ? $store.state.room.title : ($store.state.room.isTemporary ? "Temporary Room" : $store.state.room.name) }}</h1>
-        <span>{{ connectionStatus }}</span>
+      <v-layout class="room-info" column>
+        <input class="room-title" v-model="title" />
```
Comment (dyc3) [inline src/views/Room.vue]: These should use the v-input text boxes

**Comment by adapap** [comment]
Currently, this updates the title/description locally by updating the store. This should also persist on refresh by updating the database.

**Comment by dyc3** [comment]
Closing for inactivity

**Pull request closed without merging**